migrations/create-users.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      user: {
+        type: Sequelize.STRING
+      },
+      user_image_url: {
+        type: Sequelize.STRING
+      },
+      like: {
+        type: Sequelize.BOOLEAN
+      },
+      repost: {
+        type: Sequelize.BOOLEAN
+      }
+      // createdAt: {
```

Comment:
Maybe take out these comments for things you're not using.

Reply:
Fair point, and I will, thanks!